common/src/Microsoft.Azure.IIoT.Serializers.NewtonSoft/src/Microsoft.Azure.IIoT.Serializers.NewtonSoft.csproj

```diff
@@ -8,9 +8,9 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-    <PackageReference Include="JsonSchema.Net" Version="1.10.7" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\..\..\Microsoft.Azure.IIoT.Validators.JsonEverything\Microsoft.Azure.IIoT.Validators.JsonSchemaDotNet.csproj" />
     <ProjectReference Include="..\..\Microsoft.Azure.IIoT.Core\src\Microsoft.Azure.IIoT.Core.csproj" />
   </ItemGroup>
 </Project>
```

Comment:
I think we'll want a new line at the end of the file.

Reply:
I'm gonna move the project down into the correct folder at part of my next pr.

Reply:
Thanks for spotting it, even if editorconfig has insert_final_newline=true, VS seems to be ignoring it which is probably why there are many existing files without new line at the bottom of them.

Reply:
> I'm gonna move the project down into the correct folder at part of my next pr.

That's miss at my end, fixing it now.

Reply:
I got it ...

Reply:
Already made the change.

Reply:
Would you be able rename the folder from Microsoft.Azure.IIoT.Validators.JsonEverything to Microsoft.Azure.IIoT.Validators.JsonSchemaDotNet? Otherwise I can do it tomorrow.

Reply:
Done in next commit.